src/main/java/taxi/model/Driver.java

```diff
@@ -50,11 +75,21 @@ public boolean equals(Object o) {
         Driver driver = (Driver) o;
         return Objects.equals(id, driver.id)
                 && Objects.equals(name, driver.name)
-                && Objects.equals(licenseNumber, driver.licenseNumber);
+                && Objects.equals(licenseNumber, driver.licenseNumber)
+                && Objects.equals(login, driver.login)
+                && Objects.equals(password, driver.password);
```

Comment:
It's not a good practice to use user credentials anywhere outside of places where it's necessary for authorisation or authentication process